client/components/Signup/Signup.css

```diff
@@ -0,0 +1,28 @@
+.signUpForm {
+  width: 100%;
+}
+
+.home {
+    height: 100%;
+    max-width: 100%;
+    background: url(https://s3-us-west-1.amazonaws.com/beerly/beer.jpg) no-repeat center center fixed;
```

Comment:
Do we need to change the Amazon S3 image account from /beerly/ to /proudfoots-beerly/?